lib/EditListview.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import './model.dart';
+
+// Detta är andra vyn på appen
+
+class EditListview extends StatefulWidget {
+  final TodoTask task;
+
+  EditListview(this.task);
+
+  @override
+  State<StatefulWidget> createState() {
+    return EditListviewState(task);
+  }
+}
+
+class EditListviewState extends State<EditListview> {
+  String message;
+  bool status;
+
```

Comment:
Jag tror att det saknas kod här. När den skapar en ny state så måste den här klassen innehålla information kring hur den ska nyttja dina konstanter och din choiceAction för att returnera den listan du vill ha. Så här ser det ut för mig:

  final List<String> _filterList = ["All", "Done", "Not Done"];
  String _filterValue = "All";

Än en gång, ber om ursäkt att jag inte riktigt kan förklara vad jag menar, det är mycket jag inte riktigt förstår än!